styles/font.go

```diff
@@ -39,10 +39,7 @@ type Font struct { //types:add
 	// normal or small caps (inherited)
 	Variant FontVariants
 
-	// Decoration contains the bit flag [TextDecorations]
-	// (underline, line-through, etc). It must be set using
-	// [Font.SetDecoration] since it contains bit flags.
-	// It is not inherited.
+	// underline, line-through, etc (not inherited)
```

Comment:
No. The docs need to be on the field; that is all that people will actually see.